xtractmime/_patterns.py

```diff
@@ -0,0 +1,102 @@
+IMAGE_PATTERNS = (
```

Comment:
Maybe you could include a comment with a reference to the section of the standard where these tables are defined, so that it’s clear why these variables are sorted the way they are, i.e. not alphabetically.

For example:

```suggestion
# 6.1, step 1
# https://mimesniff.spec.whatwg.org/commit-snapshots/609a3a3c935fbb805b46cf3d90768d695a1dcff2/#matching-an-image-type-pattern
IMAGE_PATTERNS = (
```

This also leads me to something else: we are working with a live standard, which means it is not versioned, it keeps changing. So when including links in the documentation and code, it’s best to point to the specific snapshot of the standard that the documentation or code is referring to. That way, if the documentation or code does not match the standard in the future, it will be easier to tell whether it was a bug on our side or the fact that the standard changed.

On the living standard document you can find a link titled “Snapshot as of this commit” which will lead you to the latest snapshot of the standard. Please, use that link instead of a link to the living standard document whenever you link to the standard for reference. The only exception I can think of would be when mentioning the standard itself, instead of specific content of it, such as in the README or in the package description.